### ISSUE TITLE
Entity disappears after disconnected 

### ISSUE THREAD

**Issue by thefunkygibbon** [zachcheatham/ha-omada#31]
If i, for example disconnect a wireless device from the wifi, the plugin seems to remove the device completely from entities.  is this expected behaviour?   This causes the device to be classified as 'unavailable' rather than 'away' which i'd expect.  
Is there a way to tell the plugin to leave the entity existing should it be disconnected from the network as the chances are it'll reconnect at some point.

**Comment by rans100** [comment]
I have exactly the same problem, it started a few day ago. Don't know what update caused this.
HA behaviour: devicetracker available, person/device seen as "at Home" when the device disconnects from WiFi the device gets marked as "unvailable" and the person as "unknown". Before the device tracker stayed available and was seen as "not home".

OMADA SETUP: Omada controller v5.3.1 running on windows 10 virtual machine, omada ER605 router, EAP610 AP. 
HA SETUP: Home Assistant Core 2022.6.4, Home Assistant Supervisor 2022.05.3, Home Assistant OS 8.1.

If you need some more info, let me know.
Thanks in advance and for creating this integratiion!

**Comment by thefunkygibbon** [comment]
And just about mine 
Omada = 3 x EAP245 devices, TL-SG2218 switch, controller v5.3.1 running on docker 
HA = Home Assistant Core 2022.6.2 running on docker

**Comment by GHRRGH** [comment]
Same issue here. The entities don't automatically reappear either. Reloading the entity when they are online brings them back (until they go offline again) 

HA (2022.9.6) and Omada controller (5.5.6) both running on Docker.

**Comment by cooperaj** [comment]
Yeah. I'm seeing this too. Worth noting only a select few entities (read mobiles) are doing it. When they're removed in home assistant they're also not showing in Omada controller either so it's likely an Omada issue; my phone for instance is never removed from Omada even if I'm away for weeks, by wifes is removed after 20 minutes. 

I'd expect this integration to not remove entities at all and leave that to be a manual process maybe?

**Comment by Gatherix** [comment]
Seeing this as well, Omada hardware controller (5.6.4) - as @cooperaj mentioned, some devices persist after disconnection, others do not, haven't yet figured out what the commonality is between them or if it's just random.

@cooperaj can you clarify what you mean by them not showing up in Omada? All of my devices show up in Omada as "Known Clients", and all disconnected clients are not shown under "Clients"; as far as I understand, this is normal behavior. The devices in my HA that persist while disconnected accordingly do not show up under "Clients", as expected.

**Comment by Gatherix** [comment]
I have to do more testing, but at first glance it seems this only occurs with devices that are configured in Omada to have fixed IP addresses. After disabling custom IP for a device that previously would disappear from HA, the device now persists in HA and shows a "disconnected" status.

**Comment by Gatherix** [comment]
Making the adjustment fixed the problem for one device but not another. This problem is unsolved.

**Comment by Ayuus** [comment]
I had the same issue (refer to issue #73). Turned out that I had to make my devices (that I wanted to track) known to Omada, by manually entering a device name, in tab Config for the device in Omada Controller. Once I did that, I never lost device trackers anymore.
